client/app/shared/modals/data-upload/data-upload.component.html

```diff
@@ -57,12 +57,21 @@ <h2>Import tournament schedule</h2>
                     </tr>
                 </table>
             </div>
-
         </div>
-
-
+        
     </div>
 
-    <button id="tournament-data-close-button" (click)="clearModal()" class="close-button" data-close aria-label="Close modal" type="button">
+    <table *ngIf="teams">
```

Comment:
What about the matches? In case this is a full schedule file

Reply:
They are in lines 43-59